shell/common/engine.cc

```diff
@@ -44,6 +44,7 @@ constexpr char kAssetChannel[] = "flutter/assets";
 constexpr char kLifecycleChannel[] = "flutter/lifecycle";
 constexpr char kNavigationChannel[] = "flutter/navigation";
 constexpr char kLocalizationChannel[] = "flutter/localization";
+constexpr char kSystemChannel[] = "flutter/system";
```

Comment:
Since the memory message and the text scale message seem to get processed at two different places it might make sense to have a separate channel for the text scale after all instead of trying to separate the messages on the same system channel?

Do engine folks have an opinion on this? @cbracken @jason-simmons @Hixie

Reply:
I think if I added another channel, I would make it more general than just text scale.  Maybe "flutter/settings" or something.  Is there any place where the different channels are documented (e.g. what they are supposed to be for, what their scope should be, naming conventions, etc)?

Reply:
I don't think we have such documentation yet, but we should. Please file a bug to make sure we write such docs and add them to the API docs.

Reply:
OK, Done.  Filed [flutter/flutter#12196](https://github.com/flutter/flutter/issues/12196)

Changed the channel to "flutter/settings".